docs/src/examples/06-mmdf.jl

```diff
@@ -16,8 +16,6 @@
 
 # # Thermodynamic models
 
-using COBREXA
-
```

Comment:
this needs to be here, people will fail without copypasting this.

Reply:
it appears later on